content/en/products/products/e-briefing-app.md

```diff
@@ -5,6 +5,7 @@ description: >-
   An application to replace internal paper-based briefing binders.
 phase: alpha
 status: past
+productkind: partnership
```

Comment:
This will need to be added as a field in strapi (in the cds-website-cms repo) for these fields to persist. I should be able to do this today since I have a lighter day, but if you'd rather take a stab at it let me know and I'll make sure you have an account on the prod instance of strapi (to verify - the actual schema change is done locally in the repo).

Reply:
Ah, I didn't know that. Thanks. I wonder if there is somewhere obvious we could put a note about that in this repo.

Reply:
I do have a link to the CMS repo in the readme for this repo, but it's not explicitly clear that schema / front matter for files managed in the CMS need to be managed over there and not here. It could probably stand an iteration